vitrina/smart_contracts/models.py

```diff
@@ -0,0 +1,26 @@
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from vitrina.models import UUIDBaseModel
+
+
+def validate_markdown_file(file):
+    if not file.name.endswith('.md'):
+        raise ValidationError("Failas turi būti Markdown formato (.md).")
+
+
+class SmartContractDefaultTemplate(UUIDBaseModel):
+    default_template = models.FileField(
+        upload_to="data/files/smart_contract_default_templates",
+        verbose_name=_("Išmaniųjų sutarčių numatytasis šablonas"),
+        validators=[validate_markdown_file],
+    )
```

Comment:
Django turi atitinkamą f-iją tam:

```suggestion
    from django.core.validators import FileExtensionValidator
    
    default_template = models.FileField(
        upload_to="data/files/smart_contract_default_templates",
        verbose_name=_("Išmaniųjų sutarčių numatytasis šablonas"),
        validators=[FileExtensionValidator(allowed_extensions=["md"], message=_("Failas turi būti Markdown formato (.md)."))],
    )
```

Reply:
Dėkui